File: part02/python/numbers_processing.py
import datetime
from typing import Iterable

from pyspark import Row
from pyspark.sql import SparkSession, functions, DataFrame

from config import kafka_input_topic

spark = SparkSession.builder.master("local[*]") \
    .config('spark.jars.packages', 'org.apache.spark:spark-sql-kafka-0-10_2.12:3.4.1') \
    .getOrCreate()

File: part02/scala/src/main/scala/com/waitingforcode/NumbersProcessing.scala
package com.waitingforcode

import org.apache.spark.sql.SparkSession

object NumbersProcessing {

  def main(args: Array[String]): Unit = {
    val checkpointLocation = "/tmp/wfc/workshop/part02/checkpoint"
    val sparkSession = SparkSession.builder().master("local[*]").getOrCreate()
  }

}